aiohttp/helpers.py

```diff
@@ -746,12 +746,10 @@ def charset(self) -> Optional[str]:
     @property
     def content_length(self) -> Optional[int]:
         """The value of Content-Length HTTP header."""
-        content_length = self._headers.get(hdrs.CONTENT_LENGTH)
-
-        if content_length is not None:
-            return int(content_length)
-        else:
+        content_length = self._headers[hdrs.CONTENT_LENGTH]
+        if content_length is None:
```

Comment:
This won't be None if we don't use .get(), right?

Reply:
right in this case its typed to `str`.  Will adjust

Reply:
Little bit faster with that change as well 46931.13

Reply:
1.2% speed up to 1.8% speed up

Reply:
Also super interesting is that if we use an IP address its almost 7% faster

```
(venv) bdraco@MacBook-Pro-5 aiohttp % wrk -t 10 http://localhost:8080
Running 10s test @ http://localhost:8080
  10 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   210.76us   18.56us   1.30ms   82.25%
    Req/Sec     4.72k   106.61     5.42k    81.19%
  473971 requests in 10.10s, 71.87MB read
Requests/sec:  46931.13
Transfer/sec:      7.12MB
(venv) bdraco@MacBook-Pro-5 aiohttp % wrk -t 10 http://127.0.0.1:8080
Running 10s test @ http://127.0.0.1:8080
  10 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   195.56us   18.22us   1.04ms   82.81%
    Req/Sec     5.06k   101.73     5.40k    92.67%
  508676 requests in 10.10s, 77.13MB read
Requests/sec:  50365.54
Transfer/sec:      7.64MB
```

Reply:
Something to investigate for a future PR to see if we have something going on with names vs ips

Reply:
Not sure exactly what you're test is running, but wouldn't that just be DNS resolution or something along those lines?